ruby-challenges.rb

```diff
@@ -5,32 +5,117 @@
 
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
+
+
+# Pseudo code:
+# create def method called returnMe that uses .include to return the letter found in each word of the array
+# p each letter varibles to get the expected output, p returnMe(array, let('o')ter)
+
+def returnMe (array, letter)
+  filtered_words = array.select { |word| word.include?(letter) }
+  return filtered_words
+end
 filter_letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
+p returnMe(beverages_array, filter_letter_o)
+
+
 filter_letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
+p returnMe(beverages_array, filter_letter_t)
+
 
-# Pseudo code:
 
 # -------------------2) Create a method that takes in a hash and returns one array with all the hash values at their own index and in alphabetical order. No nested arrays. Use the test variable provided.
 # HINT: Google 'ruby get rid of nested arrays'
 
-us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Rhode Island'] }
+ us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Rhode Island'] }
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
 # Pseudo code:
+# use .flatten to convert the multiple arrays in a one dimensional array, thus removing all nested elements into a single array
+# use .sort to sort the values in the array alphabetically
+
+def sort_me_values hash
+  values = hash.values.flatten
+  sort_values = values.sort
+  return sort_values
+end
+
```

Comment:
great job!  You could refactor lines 40-42 to be just one line (just a hint - what you did is great). Also, you don't need the return. :)